examples/byovnet/variables.tf

```diff
@@ -0,0 +1,43 @@
+variable "enable_telemetry" {
+  type        = bool
+  default     = true
+  description = <<DESCRIPTION
+This variable controls whether or not telemetry is enabled for the module.
+For more information see <https://aka.ms/avm/telemetryinfo>.
+If it is set to false, then no telemetry will be collected.
+DESCRIPTION
+}
+
+variable "location" {
```

Comment:
Can we define minimum number of subnets? For example one for bastion, other for resources, third for VM's etc. There are certain best practices and we can check to include minimal subnets that we think is absolutely necessary

Reply:
As discussed, splitting this out into a different PR